lib/Dialect/FIRRTL/Transforms/ExtractInstances.cpp

```diff
@@ -166,6 +166,16 @@ struct ExtractInstancesPass
   /// create a path to the instance
   DenseMap<InnerRefAttr, InstanceOp> innerRefToInstances;
   Type stringType, pathType;
+
+  /// If set, this indicates that the `InjectDUTHierarchy` pass ran with the
+  /// `moveDut` parameter enabled.  If true, then extraction continues outside
+  /// the DUT even when extraction annotations have a wrapper module specified.
+  bool moveDut = false;
+
+  /// Set the `stopAtDut` field correctly based on the `moveDut` settings.
+  void setStopAtDUT(ExtractionInfo &info) {
+    info.stopAtDUT = !moveDut && !info.wrapperModule.empty();
+  }
```

Comment:
Could you delete `stopAtDut` in `ExtractInfo` and define `getStopAtDut(bool moveDut) { return moveDut && wrapperModule.empty();`}` instead? I think with that change you don't have to call `setStopAtDut` every ExtractInfo creation.

Reply:
Fixed in: https://github.com/llvm/circt/pull/8555/commits/b4c53873755bd1696767e89da7fef14c17b3e1c5